rasterio/plot.py

```diff
@@ -33,38 +33,69 @@
 logger = logging.getLogger(__name__)
 
 
-def show(source, cmap='gray', with_bounds=True):
+def show(source, cmap='gray', with_bounds=True, ax=None, title=None, **kwargs):
```

Comment:
True - if it can be passed through via `kwargs` and matplotlib will provide the default if you don't specifically pass one in, that seems reasonable to me.